### PR TITLE
Update Japanese i18n

### PR DESCRIPTION
**Pull request by apple502j**

### CODE REVIEW

File: i18n/ja.json [hunk 1]
```diff
@@ -5,15 +5,18 @@
...
 	},
 	"scratchlogin": "Scratchでログイン",
-	"scratchlogin-desc": "Scratchアカウントでログインする。",
-	"scratchlogin-title": "Scratchでログイン",
-	"scratchlogin-summary": "Scratchアカウントでログインします。",
+	"scratchlogin-desc": "Scratchアカウントでログインしたり、パスワードをリセットしたりする。",
 	"scratchlogin-project-id": "416761417",
-	"scratchlogin-project-author": "apple502j_sb",
+	"scratchlogin-project-author": "apple502j_sub",
 	"scratchlogin-instructions": "'''注意: Scratchでログインするには、このWikiにてすでにアカウントを持っていないといけません!'''<br/>以下の認証コードを$1認証用プロジェクト$2のコメント欄にコメントしてください。<br/>'''$3'''<br/><small>(必要な場合は、[[Special:ScratchLogin/reset|認証コードをリセットできます]]。)",
```
Comment (Kenny2github) [inline i18n/ja.json]:
```suggestion
	"scratchlogin-instructions": "'''注意: Scratchでログインするには、このWikiにてすでにアカウントを持っていないといけません!'''<br/>以下の認証コードを$1認証用プロジェクト$2のコメント欄にコメントしてください。<br/>'''$3'''<br/><small>(必要な場合は、[[Special:ScratchLogin/reset|認証コードをリセットできます]]。)</small>",
```

File: i18n/ja.json [hunk 2]
```diff
@@ -5,15 +5,18 @@
...
 	"scratchlogin-uncommented": "認証コードがコメントされていません。",
 	"scratchlogin-unregistered": "ユーザー '''$1''' はこのWikiに登録されていません。",
 	"scratchlogin-success": "'''$1'''としてログインしました。",
 	"scratchlogin-code-reset": "認証コードはリセットされました。<br/>[[Special:ScratchLogin]]へ戻る。",
```
Comment (Kenny2github) [inline i18n/ja.json]:
```suggestion
	"scratchlogin-code-reset": "認証コードはリセットされました。<br/>[[$1]]へ戻る。",
```

File: i18n/ja.json [hunk 3]
```diff
@@ -5,15 +5,18 @@
...
-	"scratchlogin-userlogin-link": "<p class=\"mw-ui-vform\" style=\"text-align: center\">[[Special:ScratchLogin|Scratchでログイン]]</p>"
+	"scratchlogin-userlogin-link": "<p class=\"mw-ui-vform\" style=\"text-align: center\">[[Special:ScratchLogin|Scratchでログイン]]</p>",
+	"scratchpasswordreset": "Scratchでパスワードリセット",
+	"scratchlogin-instructions": "以下の認証コードを$1認証用プロジェクト$2のコメント欄にコメントしてください。<br/>'''$3'''<br/><small>(必要な場合は、[[Special:ScratchLogin/reset|認証コードをリセットできます]]。)",
```
Comment (Kenny2github) [inline i18n/ja.json]:
```suggestion
	"scratchpasswordreset-instructions": "以下の認証コードを$1認証用プロジェクト$2のコメント欄にコメントしてください。<br/>'''$3'''<br/><small>(必要な場合は、[[Special:ScratchLogin/reset|認証コードをリセットできます]]。)</small>",
```

**Review by Kenny2github — CHANGES_REQUESTED** [review]
A few mistakes need correcting

**Comment by Kenny2github** [comment]
@apple502j any updates?

**Comment by T-taku** [comment]
I can fix they.
Can I help you?

**Review by Kenny2github — APPROVED** [review]

**Pull request merged**